native/ex_tokenizers/src/tokenizer.rs

```diff
@@ -125,7 +125,7 @@ fn apply_load_options(mut tokenizer: ExTokenizerImpl, options: Vec<LoadOption>)
     }
 
     if opts.disable_truncation {
-        tokenizer.with_truncation(None);
+        tokenizer.with_truncation(None).unwrap();
```

Comment:
Is API changed to return error? Why? I'm not sure `unwrap` handles it properly (mb yes)